src/main/java/org/spongepowered/api/Game.java

```diff
@@ -22,7 +22,6 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-
```

Comment:
This is what the Gradle license plugin does. There is currently no easy way around this without some really ugly hacks